resources/asia/malaysia/OSM-MY-matrix.json

```diff
@@ -5,7 +5,6 @@
   "languageCodes": ["en", "ms"],
   "name": "OpenStreetMap Malaysia Riot channel",
   "description": "All mappers are welcome! Sign up at {signupUrl}",
-  "url": "https://riot.im/app/#/room/#OpenstreetmapMalaysia:matrix.org",
-  "signupUrl": "https://riot.im/app/#/register",
```

Comment:
I think if you delete the `signupUrl` you will need to edit the description also to not include a `{signupUrl}` replacement token.

Reply:
Sorry, I missed that! I'll fix it right now.